### ISSUE TITLE
text style overrides for all emoji glyphs (outline color specifically) 

### ISSUE THREAD

**Issue by dan2bit** [renpy/renpy#5609]
Scenario:

the say text style has an outline color that doesn't match the background color. This is desirable for the standard glyphs in the font, but if emoji_font glyphs are used, the outline color is applied as a SW-NE diagonal stroke behind the emoji.

![2024-07-01 at 10 52 AM](https://github.com/renpy/renpy/assets/1513258/9ae6c1ce-03c6-482c-b85c-79453f93b8ef)

Would prefer that all emojis ignore the outline color, or that `emoji_font_` could be a style prefix that applies styles only to the emojis in a text, or some other global-ish solution.

In other words still want to be able to say
    `l_nvl "Yes actually 😅"`
    
without wrapping each emoji in a style block.

**Comment by dan2bit** [comment]
we implemented a similar solution, with `{e=😘}` for brevity, but it does defeat the purpose of easily including emoji if you have to choose between that and outline styling

**Comment by dan2bit** [comment]
from the 8.5 changelog:

> The emoji font used by Ren'Py has been updated to include 435 more glyphs, with other glyphs having been updated. (This includes most Unicode 16 emoji, but not unicode 17 emoji.)

does that mean more appetite for fixing this issue with outline styles being inappropriately applied to emoji?
